src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,97 @@
+import Text from "src/components/Text";
+import React, { useState } from "react";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+
+import * as S from "./styles";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+    const [name, setName] = useState<string>("");
+    const [subject, setSubject] = useState<string>("");
+    const [content, setContent] = useState<string>("");
+  
+    const handleSubmit = (event: React.MouseEvent) => {
+        event.preventDefault();
+  
+        if (!name || !subject || !content) {
+            return;
+        }
+  
+        const body =
+            `Hi Nixon,\n\n` +
+            `${content}\n\n` +
+            `Thank you.\n\n` +
+            `Best regards,\n\n` +
+            `${name}\n`;
+  
+        window.open(
```

Comment:
What's the difference between `window.open` and `location.href`

Reply:
window.open buat open a new window ato tab, location.href buat set or return the complete URL of the current page